api/test/integration/test_tasks_endpoints.tavern.yaml

```diff
@@ -0,0 +1,97 @@
+---
+test_name: PUT /agents/upgrade
+
+marks:
+  - base_tests
+
+stages:
+
+  - name: Generate two task IDs
+    request:
+      verify: False
+      url: "{protocol:s}://{host:s}:{port:d}/agents/upgrade"
+      method: PUT
+      headers:
+        Authorization: "Bearer {test_login_token}"
+      params:
+        agents_list: 001,002
+        wpk_repo: packages.wazuh.com/wpk/
+        version: 3.13.1
+        force: true
+    response:
+      status_code: 200
+
+---
+test_name: GET /tasks/status
+
+stages:
+
+  - name: Missing task IDs
```

Comment:
Why are we not giving back all tasks if nothing is specified? Does the task socket accept such petition?